integrationTests/CampaignRepositoryTest.php

```diff
@@ -65,13 +67,15 @@ public function testItFindsNo10MonthOldCampaignEvenIfCharityAwaitingGiftAidAppro
         $campaign = new Campaign(
             Salesforce18Id::ofCampaign('xxxxxxxxxxxxxxxxxx'),
             $this->getCharityAwaitingGiftAidApproval(),
-            startDate: new \DateTimeImmutable('-9 months'),
+            startDate: new \DateTimeImmutable('-11 months'),
```

Comment:
added check in Campaign model to make sure a campaign can't start before it ends. I think that probably makes sense but there's an argument that any checks like that should be done in SF if needed and by the time the data gets to matchbot its too late and we should just accept whatever SF gives us, since we can't make the exception bubble up to the UI.